src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -2764,7 +2764,7 @@ void executeOperationsActionsByKind(ImmutableArray<IOperation> operationsToAnaly
                 }
             }
 
-            void executeOperationsBlockActions(ImmutableArray<IOperation> operationBlocksToAnalyze, ImmutableArray<IOperation> operationsToAnalyze, IEnumerable<ExecutableCodeBlockAnalyzerActions> codeBlockActions)
+            void executeOperationsBlockActions(ImmutableArray<IOperation> operationBlocksToAnalyze, ImmutableArray<IOperation> operationsToAnalyze, ArrayBuilder<ExecutableCodeBlockAnalyzerActions> codeBlockActions)
```

Comment:
making these more strongly typed, when possible, helps a lot as it means that iteration is alloc free.

Reply:
IEnumerable is now gone from AnalyzerDriver and ANalyzerExecutor. (except for executing suppressions, which is such a rare case i dont' care).